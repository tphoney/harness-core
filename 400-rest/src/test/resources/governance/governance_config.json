{
  "accountId": "kmpySmUISimoRrJL6NL73w",
  "deploymentFreeze": true,
  "timeRangeBasedFreezeConfigs": [
    {
      "uuid": "FREEZE_ID",
      "freezeForAllApps": false,
      "name": "freezer",
      "description": "Holiday production freeze",
      "applicable": true,
      "userGroups": [
        "aaOLwzRsTMOoSlDrjstIHQ",
        "BH5_gD_VRDaWcSCsnZKSug"
      ],
      "appIds": [
        "aaOLwzRsTMOoSlDrjstIHQ",
        "BH5_gD_VRDaWcSCsnZKSug"
      ],
      "environmentTypes": [
        "PROD"
      ],
      "appSelections": [
        {
          "filterType": "CUSTOM",
          "apps": [
            "aaOLwzRsTMOoSlDrjstIHQ"
          ],
          "envSelection": {
            "filterType": "CUSTOM",
            "environments": ["env1", "env2"]
          },
          "serviceSelection": {
            "filterType": "CUSTOM",
            "services": ["service1", "service2"]
          }
        },
        {
          "filterType": "ALL",
          "envSelection": {
            "filterType": "ALL_NON_PROD"
          }
        }
      ],
      "timeRange": {
        "durationBased": false,
        "from": 1609372800000,
        "to": 1609459200000,
        "timeZone": "Africa/Addis_Ababa"
      }
    }
  ],
  "uuid": "GOVERNANCE_CONFIG_ID",
  "weeklyFreezeConfigs": []
}