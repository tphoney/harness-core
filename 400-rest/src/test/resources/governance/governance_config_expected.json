{
  "uuid":"GOVERNANCE_CONFIG_ID",
  "accountId":"kmpySmUISimoRrJL6NL73w",
  "appId":"__GLOBAL_APP_ID__",
  "deploymentFreeze":true,
  "timeRangeBasedFreezeConfigs":[
    {
      "freezeForAllApps":false,
      "appIds":[
        "aaOLwzRsTMOoSlDrjstIHQ",
        "BH5_gD_VRDaWcSCsnZKSug"
      ],
      "environmentTypes":[
        "PROD"
      ],
      "timeRange":{
        "from":1609372800000,
        "to":1609459200000,
        "timeZone":"Africa/Addis_Ababa",
        "durationBased": false,
        "expires": false
      },
      "name":"freezer",
      "description":"Holiday production freeze",
      "applicable":false,
      "appSelections":[
        {
          "filterType":"CUSTOM",
          "envSelection":{
            "filterType":"CUSTOM",
            "environments":[
              "env1",
              "env2"
            ]
          },
          "apps":[
            "aaOLwzRsTMOoSlDrjstIHQ"
          ],
          "serviceSelection": {
            "filterType": "CUSTOM",
            "services": ["service1", "service2"]
          }
        },
        {
          "filterType":"ALL",
          "envSelection":{
            "filterType":"ALL_NON_PROD"
          }
        }
      ],
      "userGroups":[
        "aaOLwzRsTMOoSlDrjstIHQ",
        "BH5_gD_VRDaWcSCsnZKSug"
      ],
      "uuid":"FREEZE_ID",
      "freezeWindowState":"INACTIVE"
    }
  ],
  "enableNextIterations":false,
  "enableNextCloseIterations":false
}